governance/polls/Proposal - add WBTC-B vault type - January 11, 2021.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Should we add a new WBTC vault type with lower LR? - January 11, 2021
+summary: This poll will provide guidance to the Domain Teams for creating a new WBTC based vault type
+discussion_link: https://forum.makerdao.com/t/signal-request-new-iam-vault-type-for-wbtc-with-lower-lr/5736/
+vote_type: Plurality Voting
+options:
+   0: Abstain
+   1: Yes
+   2: No
+---
+# Poll: Should we add a new WBTC vault type with lower LR? - January 11, 2021
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the Maker Community. The community can vote in this poll to express support or opposition to add a new WBTC-based vault type with a lower Liquidation Ratio - similar to ETH-B.
+
+This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for three days beginning on Monday, January 11th at 16:00 UTC.
+
+**This is a binary vote. In a binary vote, you may vote for a single option. You should vote for the option which you prefer. Alternatively, if you would accept either of the two options and have no preference between them, you should vote 'Abstain'.**
+
+## Review
+
```

Comment:
I'd add a little more detail here as to the similarity of this to greenlight. I'm happy to fill this in later though, so feel free to leave as is.